### PR TITLE
Ac frontend

### PR DESCRIPTION
**Pull request by acordero4852**
Commented the console.log and added a feature to prevent the user's account from logging out when the page refreshes, plus completed most feature from the milestone.

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -19,15 +19,26 @@ function App() {
...
-    id: 0,
-    firstName: '',
-    lastName: '',
-    email: '',
-    phoneNumber: '',
-  });
+  const [userInfo, setUserInfo] = useState({});
   const [footer, setFooter] = useState(true);
 
+  console.log(userInfo)
```
Comment (LifeBringer) [inline src/components/App/App.jsx]: Please comment out or delete

File: src/components/App/App.jsx [hunk 2]
```diff
@@ -19,15 +19,26 @@ function App() {
...
+  console.log(isAuthenticated)
```
Comment (LifeBringer) [inline src/components/App/App.jsx]: Please comment out or delete

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comments

File: src/components/App/App.jsx (hunk 2 above)
Comment (acordero4852) [reply src/components/App/App.jsx]: commented the console logs

**Review by LifeBringer — APPROVED** [review]

**Pull request merged**